### PR TITLE
Convert two css-counter (t1202-counter-*) tests into reftests

### PR DESCRIPTION
**Pull request by Ahmad-S792**
#### 92e26bad7aef126178d279d5fd326ef7bec9852b
<pre>
Convert two css-counter (t1202-counter-*) tests into reftests

<a href="https://bugs.webkit.org/show_bug.cgi?id=266850">https://bugs.webkit.org/show_bug.cgi?id=266850</a>

Reviewed by Tim Nguyen.

Merge: <a href="https://chromium.googlesource.com/chromium/blink/+/14b824be9c72c15d1ec55d656eb63451f92cad5d">https://chromium.googlesource.com/chromium/blink/+/14b824be9c72c15d1ec55d656eb63451f92cad5d</a>

This PR imports test expectations from Blink / Chromium to change two tests into &apos;reftests&apos;.

* LayoutTests/css2.1/t1202-counter-09-b-expected.html: New Expectation
* LayoutTests/css2.1/t1202-counters-09-b-expected.html: Ditto

&gt; Remove Platform Specific Files:
* LayoutTests/platform/glib/css2.1/t1202-counter-09-b-expected.txt:
* LayoutTests/platform/glib/css2.1/t1202-counters-09-b-expected.txt:
* LayoutTests/platform/gtk/css2.1/t1202-counter-09-b-expected.png:
* LayoutTests/platform/gtk/css2.1/t1202-counters-09-b-expected.png:
* LayoutTests/platform/ios/css2.1/t1202-counter-09-b-expected.txt:
* LayoutTests/platform/ios/css2.1/t1202-counters-09-b-expected.txt:
* LayoutTests/platform/mac/css2.1/t1202-counter-09-b-expected.png:
* LayoutTests/platform/mac/css2.1/t1202-counter-09-b-expected.txt:
* LayoutTests/platform/mac/css2.1/t1202-counters-09-b-expected.png:
* LayoutTests/platform/mac/css2.1/t1202-counters-09-b-expected.txt:
* LayoutTests/platform/wincairo/css2.1/t1202-counter-09-b-expected.txt:
* LayoutTests/platform/wincairo/css2.1/t1202-counters-09-b-expected.txt:

Canonical link: <a href="https://commits.webkit.org/272483@main">https://commits.webkit.org/272483@main</a>
</pre>

https://github.com/WebKit/WebKit/commit/796bd79c97b7dd786988fba156760ab6a192facc

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/31792 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/10483 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/33530 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/34293 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/28790 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/12843 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/7725 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/28386 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/32154 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/8844 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/28389 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/7635 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/7805 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/28302 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/35640 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/28908 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/28754 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/33917 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/7894 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/50/builds/5892 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/31776 "Passed tests") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/9548 "Built successfully") | | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/7451 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/8565 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/8416 "Built successfully") | | |

### CODE REVIEW

File: LayoutTests/css2.1/t1202-counter-09-b-expected.html [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  <title>CSS 2.1 Test Suite: content: counter(c, georgian)</title>
+  <link rel="help" href="http://www.w3.org/TR/CSS21/generate.html#propdef-content">
+  <link rel="help" href="http://www.w3.org/TR/CSS21/syndata.html#counter">
+  <link rel="help" href="http://www.w3.org/TR/CSS21/generate.html#counter-styles">
+  <meta http-equiv="Content-Style-Type" content="text/css"></meta>
+  <style type="text/css">
+  body { white-space: nowrap; }
+  #test { counter-reset: c; }
+  #test span { counter-increment: c; }
+  #test span:before { content: counter(c, georgian); }
```
Comment (nt1m) [inline LayoutTests/css2.1/t1202-counter-09-b-expected.html]: These styles are unused in this test

File: LayoutTests/css2.1/t1202-counter-09-b-expected.html [hunk 2]
```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN">
+<html>
+ <head>
+  <title>CSS 2.1 Test Suite: content: counter(c, georgian)</title>
+  <link rel="help" href="http://www.w3.org/TR/CSS21/generate.html#propdef-content">
+  <link rel="help" href="http://www.w3.org/TR/CSS21/syndata.html#counter">
+  <link rel="help" href="http://www.w3.org/TR/CSS21/generate.html#counter-styles">
+  <meta http-equiv="Content-Style-Type" content="text/css"></meta>
+  <style type="text/css">
```
Comment (nt1m) [inline LayoutTests/css2.1/t1202-counter-09-b-expected.html]:
```suggestion
  <style>
```

File: LayoutTests/css2.1/t1202-counters-09-b-expected.html [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+  <title>CSS 2.1 Test Suite: content: counters(c, &quot;.", georgian)</title>
+  <link rel="help" href="http://www.w3.org/TR/CSS21/generate.html#propdef-content">
+  <link rel="help" href="http://www.w3.org/TR/CSS21/syndata.html#counter">
+  <link rel="help" href="http://www.w3.org/TR/CSS21/generate.html#counter-styles">
+  <meta http-equiv="Content-Style-Type" content="text/css"></meta>
+  <style type="text/css">
+  body { white-space: nowrap; overflow: hidden; }
+  body, #test span:first-child { counter-reset: c; }
+  p, #test span { counter-increment: c; }
+  #test span:before { content: counters(c, ".", georgian); }
```
Comment (nt1m) [inline LayoutTests/css2.1/t1202-counters-09-b-expected.html]: These are also unused.

File: LayoutTests/css2.1/t1202-counters-09-b-expected.html [hunk 4]
```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN">
+<html>
+ <head>
+  <title>CSS 2.1 Test Suite: content: counters(c, &quot;.", georgian)</title>
+  <link rel="help" href="http://www.w3.org/TR/CSS21/generate.html#propdef-content">
+  <link rel="help" href="http://www.w3.org/TR/CSS21/syndata.html#counter">
+  <link rel="help" href="http://www.w3.org/TR/CSS21/generate.html#counter-styles">
+  <meta http-equiv="Content-Style-Type" content="text/css"></meta>
+  <style type="text/css">
```
Comment (nt1m) [inline LayoutTests/css2.1/t1202-counters-09-b-expected.html]:
```suggestion
  <style>
```

File: LayoutTests/css2.1/t1202-counter-09-b-expected.html [hunk 5]
```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN">
+<html>
+ <head>
+  <title>CSS 2.1 Test Suite: content: counter(c, georgian)</title>
+  <link rel="help" href="http://www.w3.org/TR/CSS21/generate.html#propdef-content">
+  <link rel="help" href="http://www.w3.org/TR/CSS21/syndata.html#counter">
+  <link rel="help" href="http://www.w3.org/TR/CSS21/generate.html#counter-styles">
+  <meta http-equiv="Content-Style-Type" content="text/css"></meta>
```
Comment (nt1m) [inline LayoutTests/css2.1/t1202-counter-09-b-expected.html]: You can remove this line.

File: LayoutTests/css2.1/t1202-counter-09-b-expected.html [hunk 6]
```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN">
+<html>
+ <head>
+  <title>CSS 2.1 Test Suite: content: counter(c, georgian)</title>
```
Comment (nt1m) [inline LayoutTests/css2.1/t1202-counter-09-b-expected.html]:
```suggestion
  <title>Test reference: content: counter(c, georgian)</title>
```

File: LayoutTests/css2.1/t1202-counters-09-b-expected.html [hunk 7]
```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN">
+<html>
+ <head>
+  <title>CSS 2.1 Test Suite: content: counters(c, &quot;.", georgian)</title>
```
Comment (nt1m) [inline LayoutTests/css2.1/t1202-counters-09-b-expected.html]:
```suggestion
  <title>Test reference: content: counters(c, &quot;.", georgian)</title>
```

File: LayoutTests/css2.1/t1202-counters-09-b-expected.html [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+  <link rel="help" href="http://www.w3.org/TR/CSS21/generate.html#propdef-content">
+  <link rel="help" href="http://www.w3.org/TR/CSS21/syndata.html#counter">
+  <link rel="help" href="http://www.w3.org/TR/CSS21/generate.html#counter-styles">
+  <meta http-equiv="Content-Style-Type" content="text/css"></meta>
```
Comment (nt1m) [inline LayoutTests/css2.1/t1202-counters-09-b-expected.html]: You can remove this line.

**Review by nt1m — APPROVED** [review]

**Comment by nt1m** [comment]
@Ahmad-S792 I think the tests are failing because the references are missing `<meta charset="utf-8">`

File: LayoutTests/css2.1/t1202-counters-09-b-expected.html (hunk 3 above)
Comment (Ahmad-S792) [reply LayoutTests/css2.1/t1202-counters-09-b-expected.html]: Done in [e2be840](https://github.com/WebKit/WebKit/commit/e2be84005a836155a72ee3465daaba6c17546dc9).

File: LayoutTests/css2.1/t1202-counters-09-b-expected.html (hunk 4 above)
Comment (Ahmad-S792) [reply LayoutTests/css2.1/t1202-counters-09-b-expected.html]: Done in [e2be840](https://github.com/WebKit/WebKit/commit/e2be84005a836155a72ee3465daaba6c17546dc9).

File: LayoutTests/css2.1/t1202-counter-09-b-expected.html (hunk 5 above)
Comment (Ahmad-S792) [reply LayoutTests/css2.1/t1202-counter-09-b-expected.html]: Done in [e2be840](https://github.com/WebKit/WebKit/commit/e2be84005a836155a72ee3465daaba6c17546dc9).

File: LayoutTests/css2.1/t1202-counters-09-b-expected.html (hunk 7 above)
Comment (Ahmad-S792) [reply LayoutTests/css2.1/t1202-counters-09-b-expected.html]: Done in [e2be840](https://github.com/WebKit/WebKit/commit/e2be84005a836155a72ee3465daaba6c17546dc9).

File: LayoutTests/css2.1/t1202-counters-09-b-expected.html (hunk 8 above)
Comment (Ahmad-S792) [reply LayoutTests/css2.1/t1202-counters-09-b-expected.html]: Done in [e2be840](https://github.com/WebKit/WebKit/commit/e2be84005a836155a72ee3465daaba6c17546dc9).

File: LayoutTests/css2.1/t1202-counter-09-b-expected.html (hunk 6 above)
Comment (Ahmad-S792) [reply LayoutTests/css2.1/t1202-counter-09-b-expected.html]: Done in [e2be840](https://github.com/WebKit/WebKit/commit/e2be84005a836155a72ee3465daaba6c17546dc9).

File: LayoutTests/css2.1/t1202-counter-09-b-expected.html (hunk 2 above)
Comment (Ahmad-S792) [reply LayoutTests/css2.1/t1202-counter-09-b-expected.html]: Done in [e2be840](https://github.com/WebKit/WebKit/commit/e2be84005a836155a72ee3465daaba6c17546dc9).

File: LayoutTests/css2.1/t1202-counter-09-b-expected.html (hunk 1 above)
Comment (Ahmad-S792) [reply LayoutTests/css2.1/t1202-counter-09-b-expected.html]: Done in [e2be840](https://github.com/WebKit/WebKit/commit/e2be84005a836155a72ee3465daaba6c17546dc9).

**Comment by Ahmad-S792** [comment]
> @Ahmad-S792 I think the tests are failing because the references are missing `<meta charset="utf-8">`

Done in [e2be840](https://github.com/WebKit/WebKit/commit/e2be84005a836155a72ee3465daaba6c17546dc9).

File: LayoutTests/css2.1/t1202-counters-09-b-expected.html [hunk 9]
```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html>
+ <head>
+  <title>Test reference: content: counters(c, &quot;.", georgian)</title>
+  <meta charset="utf-8">
+  <link rel="help" href="http://www.w3.org/TR/CSS21/generate.html#propdef-content">
+  <link rel="help" href="http://www.w3.org/TR/CSS21/syndata.html#counter">
+  <link rel="help" href="http://www.w3.org/TR/CSS21/generate.html#counter-styles">
+  <style>
+  body { white-space: nowrap; overflow: hidden; }
```
Comment (nt1m) [inline LayoutTests/css2.1/t1202-counters-09-b-expected.html]: The test file doesn't have `overflow: hidden` hence the scrollbar difference. You can either add it to the test file or remove it from here.

Comment (Ahmad-S792) [reply LayoutTests/css2.1/t1202-counters-09-b-expected.html]: Thanks!

**Pull request merged**